modules/library/referencing/src/test/java/org/geotools/referencing/proj/gt_proj_compare.py

```diff
@@ -0,0 +1,126 @@
+import csv
```

Comment:
Is this a leftover? We need tests written in Java, that will actually run. A python script to generate sample data is welcomed, it's a one-off, but a tool that performing checks in python is a no-go

Reply:
That's not the testing tool. That's the python script that allows to compare the GeoTools generated PROJs with the official PROJs String so you can easily spot the differences and eventually customize the refinements file, tweaking/adding the rules.
I have mentioned it in the README too.